cmd/gh/main_test.go

```diff
@@ -76,3 +88,128 @@ check your internet connection or https://githubstatus.com
 		})
 	}
 }
+
+func Test_IntegrationRepoFork(t *testing.T) {
+	// Set up temp dir
+	tempDir := t.TempDir()
+	oldWd, _ := os.Getwd()
+	assert.NoError(t, os.Chdir(tempDir))
+	t.Cleanup(func() { _ = os.Chdir(oldWd) })
+
+	// Set up server to handle network requests
+	s := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		path := r.URL.Path
+		gitPath := regexp.MustCompile(`\/someone\/repo\.git\/.*`)
+		forkPath := "/repos/owner/repo/forks"
```

Comment:
If the `gh` process never makes a POST to this endpoint, does the test fail?

Reply:
Not yet, but that functionality could be added. I was thinking once this work is finished that it would resemble `httpmock` with some sort of verify function that would do that.